video_xblock/backends/brightcove.py

```diff
@@ -612,6 +612,6 @@ def download_default_transcript(self, url=None, language_code=None):  # pylint:
         if url is None:
             raise VideoXBlockException(_('`url` parameter is required.'))
         data = requests.get(url)
-        text = data.content.decode('utf8')
-        cleaned_captions_text = remove_escaping(text)
-        return unicode(cleaned_captions_text)
+        cleaned_captions_text = remove_escaping(data.content)
```

Comment:
Just a nit - defined `cleaned_captions_text` used once, I would `return` in L615:

```suggestion
        return  remove_escaping(data.content)
```

Reply:
agree, done